husky_components/tabs/main.js

```diff
@@ -54,7 +55,8 @@ define(function() {
             preSelectEvent: {
                 enabled: false,
                 triggerSelectItem: true
-            }
+            },
+            tabChangeAllowed: null
```

Comment:
@ReneZumtobel That would have been the crucial part: Moving that logic to the other component. However, it might be a better idea to change this to a callback which tells the tabs only if the change is allowed or not, and create the overlay in here. Would have the advantage that we could easily reuse it for other forms containing tabs.